translator/trace/zipkinv1/json.go

```diff
@@ -39,14 +41,30 @@ var (
 	msgZipkinV1SpanIDError        = "zipkinV1 span id"
 	msgZipkinV1ParentIDError      = "zipkinV1 span parentId"
 	// Generic hex to ID conversion errors
-	errHexTraceIDWrongLen = errors.New("hex traceId span has wrong length (expected 16 or 32)")
-	errHexTraceIDParsing  = errors.New("failed to parse hex traceId")
-	errHexTraceIDZero     = errors.New("traceId is zero")
-	errHexIDWrongLen      = errors.New("hex Id has wrong length (expected 16)")
-	errHexIDParsing       = errors.New("failed to parse hex Id")
-	errHexIDZero          = errors.New("ID is zero")
+	errHexTraceIDWrongLen                     = errors.New("hex traceId span has wrong length (expected 16 or 32)")
+	errHexTraceIDParsing                      = errors.New("failed to parse hex traceId")
+	errHexTraceIDZero                         = errors.New("traceId is zero")
+	errHexIDWrongLen                          = errors.New("hex Id has wrong length (expected 16)")
+	errHexIDParsing                           = errors.New("failed to parse hex Id")
+	errHexIDZero                              = errors.New("ID is zero")
+	_                     model.TracesDecoder = (*jsonDecoder)(nil)
 )
 
+type jsonDecoder struct {
+	// ParseStringTags should be set to true if tags should be converted to numbers when possible.
+	ParseStringTags bool
+}
+
+// DecodeTraces from JSON bytes.
+func (j jsonDecoder) DecodeTraces(buf []byte) (interface{}, error) {
```

Comment:
(Would be nice to stop using OC as intermediate representation, i.e. not related to this PR :) )